src/widgets/DataTable/HeaderCell.js

```diff
@@ -14,6 +16,8 @@ import { TouchableOpacity, Text, StyleSheet, View, TouchableOpacityPropTypes } f
  * @prop {ReactElement} SortAscComponent      Icon component for ascending sorting
  * @prop {ReactElement} SortDescComponent     Icon component for descending sorting
  * @prop {ReactElement} SortNeutralComponent  Icon component for neutral state, no sort.
+ * @prop {Object}       containerStyle        Style object for the wrapping Touchable or View.
+ * @prop {Object}       textStyle             Style object for the inner text component.
```

Comment:
Hope I'm not too annoying
```suggestion
 * @prop {Object}       textStyle             Style object for the inner Text component.
```

Reply:
587bfcd3594d21a1a3a206d05cdcc7c1d1499773

Note to self: commit all the suggestions first :(